README.md

```diff
@@ -1,4 +1,262 @@
 IPFS RFCs
 =====
 
-This repository will be the place where people propose, comment on, and refer to RFCs for changes to IPFS and its surrounding projects. Its initial manifestation will probably look a lot like the [Rust lang RFCs repository](https://github.com/rust-lang/rfcs)
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#ipfs-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+IPFS, libp2p or the RFC process itself. What constitutes a
```

Comment:
Is “libp2p” correct here? RFC 0001 talks about replicating this process over in the libp2p org, so I would assume the process *here* is not the right one for libp2p. Or are we going to do it all here for now until we decide it makes sense to replicate all this in the other orgs?

Reply:
until there's a libp2p org, with people to run it, these RFCs apply to both.

Reply:
👍 sounds good; just wanted to make sure.